delta/kernel/src/test/scala/ch/epfl/bluebrain/nexus/delta/kernel/utils/IOUtilsSuite.scala

```diff
@@ -0,0 +1,33 @@
+package ch.epfl.bluebrain.nexus.delta.kernel.utils
+
+import cats.effect.{Clock, IO}
+import cats.implicits.toFlatMapOps
+import ch.epfl.bluebrain.nexus.delta.kernel.utils.IOUtils.IoOps
+import munit.FunSuite
+
+import java.time.Instant
+import scala.concurrent.duration._
+
+class IOUtilsSuite extends FunSuite {
```

Comment:
We can use a CatsEffectsSuite here

Reply:
We can't because it's not visible in `kernel`